src/test/java/HomeWork17.java

```diff
@@ -0,0 +1,24 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class HomeWork17 extends BaseTest {
+    @Test
+    public void addSongToPlaylist() {
+
```

Comment:
Your homework is incomplete. Please complete your homework and resubmit it. Please look at the details in LMS for homework17.